_scripts/tutorialformatter.py

```diff
@@ -118,7 +118,7 @@ def flatten_sub_tutorials(self, file_):
                 if sub_name in subs:
                     flattened_lines.extend( subs[sub_name] )
                 else:
-                    print 'tutorialformatter.py error: sub-tutorial %s not found.' % sub_name
+                    print ('tutorialformatter.py error: sub-tutorial %s not found.' % sub_name)
```

Comment:
```suggestion
                    print('tutorialformatter.py error: sub-tutorial " + sub_name +  " not found.')
```
Removes the space before the bracket and the C-style string formatting (which belongs on a dump imo).

Also remove the space before the bracket in the other line please